cumulus/ansible/README.md

```diff
@@ -22,17 +22,20 @@
 
 ```json
     {"name": "test cluster",
-    "type": "ansible",
-	"playbook": "sge",
-	"cluster_config": {
-         "master_instance_type": "t2.micro",
-         "master_instance_ami": "ami-03de3c63",
-         "node_instance_count": 2,
-         "node_instance_type": "t2.micro",
-         "node_instance_ami": "ami-03de3c63",
-         "ansible_ssh_user": "ubuntu"
+     "type": "ansible",
+     "config": {
+       "launch": {
+         "spec": "sge",
```

Comment:
Make sense. However, I think it might be best to rename default.yml to ec2.yml. Then use the group_by module to "rename" the the groups to the once the girdengine playbook expects. The reasoning behind this is that the group names used in sge.yml are a little specfic to sge.